apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -106,13 +106,8 @@ public function insert_default_data() {
 			return;
 		}
 
-		if ( ! $this->wp_template_inserter->is_template_data_inserted() ) {
-			$this->wp_template_inserter->insert_default_template_data();
-		}
-
-		if ( ! $this->wp_template_inserter->is_pages_data_inserted() ) {
-			$this->wp_template_inserter->insert_default_pages();
-		}
+		$this->wp_template_inserter->insert_default_template_data();
+		$this->wp_template_inserter->insert_default_pages();
```

Comment:
These checks are not needed here since they are duplicated inside of the functions being called below.
Furthermore, baling here would prevent proper execution of action hooks that we introduced for logging:
https://github.com/Automattic/wp-calypso/blob/babd1b81a044b7412d0a96940014baaf62604a1a/apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template-inserter.php#L279-L288